update-bot/wlmbots/lib/template_checker.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, config=None):
             config = {}
         self.tpl_match_regex = None
         self.config = config
+        self._config = config
```

Comment:
Good question - no rubbish - and {} sounds as well better to me. I guess _config is used somwhere in pywikibot? @gbirke